striatum/storage/history.py

```diff
@@ -132,6 +132,7 @@ def add_history(self, context, action, reward=None):
         -----
         """
         action_time = datetime.now()
+        history_id = self.n_histories
         if reward is None:
             history = History(self.n_histories, action_time, context, action)
             self.unrewarded_histories[history.history_id] = history
```

Comment:
If you want to define history_id first, it would be better to use `history_id` instead of `self.n_histories` or `history.history_id` in line 137, 138.